### PR TITLE
Cockpit 279

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-279.html

### CODE REVIEW

**Comment by jelly** [comment]
I pushed the release links, everything should be ready for review now.

**Comment by garrett** [comment]
I wrote most of the notes and reviewed+rewrote what I didn't write. It would be nice if someone other than me can review it this time around please. :grin: Thanks!

**Review by jelly — CHANGES_REQUESTED** [review]

File: _posts/2022-11-03-cockpit-279.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Cockpit 279
+author: garrett
+date: '2022-11-03'
+tags: cockpit, machines
+slug: cockpit-279
+category: release
+summary: ''
```
[the remark this answers is not in the document]
Comment (jelly) [reply _posts/2022-11-03-cockpit-279.md:8]:
```suggestion
summary: 'Dark mode support, insert and eject CD & DVD media, and allow trim for disks in machines'
```

**Review by jelly — CHANGES_REQUESTED** [review]

File: _posts/2022-11-03-cockpit-279.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (garrett) [reply _posts/2022-11-03-cockpit-279.md:8]: What about this?

```suggestion
summary: 'Dark mode, insert and eject disc media for VMs, and allow trim on VM disks'
```

[the remark this answers is not in the document]
Comment (jelly) [reply _posts/2022-11-03-cockpit-279.md:8]: sounds better!

**Review by jelly — APPROVED** [review]

**Pull request merged**